### PR TITLE
Lazyload

### PR DESCRIPTION
**Pull request by mwernimont**
Changes made:
-----------
Description

Section Title now are lazy loaded with the intersection observer api

With the new small images loading so quickly they would appear during the initial load and then disappear to their rightful place after the splash page was done loading.  I have set it up so all the components wait for the splash to load and then they render in after.  A loading page might help with the dumb USGS header footer awkwardness.

I also removed the Webpack prefetch from most of the components as if we prefetch all of them its basically like we aren't prefetching any of them.  I was the one who set up that, so just correcting my mistake.  

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [x] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/views/Visualization.vue [hunk 1]
```diff
@@ -109,6 +123,28 @@ export default {
...
+          entry.target.nextElementSibling.srcset = entry.target.dataset.srcset; 
+          //stop observing img after the switch
+          observer.unobserve(entry.target);
+        }
+        const imgTargets = document.querySelectorAll(".lazy > source");
+        const imgObserver = new IntersectionObserver(loadImg, {
+          //Watch entire viewport
+          root: null,
+          threshold: 0,
+          rootMargin: "300px"
```
Comment (collnell) [inline src/views/Visualization.vue:144]: So `rootMargin` is where the trigger is located? Is this in view or off the page?

**Review by collnell — APPROVED** [review]
This is great. Aaron told me this was possible a long time ago but I was yet to figure it out myself. It looks pretty smooth, doesn't seem like we need webpack for the first map even, or anything else later on 👍🏼

File: src/views/Visualization.vue (hunk 1 above)
Comment (mwernimont) [reply src/views/Visualization.vue:144]: Yep, so the threshold 0 is the very top of the observed element and I am saying 300px before that load in the image.

Comment (mwernimont) [reply src/views/Visualization.vue:144]: I had it set to 200px initially but mobile works better with a larger margin.

**Pull request merged**